src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/genericwhitespace/InputGenericWhitespaceDefault.java

```diff
@@ -63,10 +63,10 @@ public int getConstructor(Class<?>... parameterTypes)
     public static class IntEnumValueType<E extends Enum<E> & IntEnum> {
     }
 
-    public static class IntEnumValueType2<E extends Enum<E>& IntEnum> { // violation
+    class IntEnumValueType2<E extends Enum<E>& IntEnum> { // violation ''&' is not preceded .*'
```

Comment:
IntEnumValueType2 -> IntEVT2
IntEnum -> IntE

please do all to give all space to message.

This check does not care about types names, even human who read - do not care.

Reply:
Done